### ISSUE TITLE
Exposing fetchImage to prefetch image

### ISSUE THREAD

**Issue by luatnd** [crelies/RemoteImage#17]
Hi, 
My app need to prefetch image and store it in cache.
After that we display the image later.

I tried to 
```
let url:String = "https://..../image.jpg"
let imageType = RemoteImageType.url(URL(string: url)!)
let service = RemoteImageServiceFactory.makeRemoteImageService()
service.fetchImage(ofType: imageType)
```
But xCode notice me:
> 'fetchImage' is inaccessible due to 'internal' protection level

Can we expose the `fetchImage` func without leading to any issue?

**Comment by crelies** [comment]
Hey,

thanks for your feedback. I like your idea. I just published version `4.0.0`. You will like it ;)

**Comment by luatnd** [comment]
Many thanks, love your work 👍
